### DISCUSSION TITLE
Rapid Movement Override

### DISCUSSION THREAD

**Discussion by Monberg75** [winder/Universal-G-Code-Sender#2540] [Ideas]
Fusion (free version) set the rapid movement rate as the same as feed rate. 

It would be nice to be able to override the rapid movement with something faster, but I can't set the existing override higher than 100%, so it's only slowing everything down.

Maybe it would be possible to set it to like 150% og maybe up to 200% in future, so that we would have the 'correct' behaviour of the machine making jobs faster, like the other overrides of spindle and feedrates.

**Comment by breiler** [comment]
See the reply here:
https://github.com/winder/Universal-G-Code-Sender/discussions/2468#discussioncomment-9717438
